app/models/edition.rb

```diff
@@ -4,7 +4,7 @@ class Edition < ActiveRecord::Base
   belongs_to :guide, touch: true
   belongs_to :user
 
-  has_many :approvals
+  has_one :approval
```

Comment:
We only had one approval per edition already, so it works nicely without migrating any data.